src/transport/SessionManager.cpp

```diff
@@ -582,6 +589,79 @@ void SessionManager::OnMessageReceived(const PeerAddress & peerAddress, System::
     }
 }
 
+#if CHIP_CONFIG_TCP_SUPPORT_ENABLED
+void SessionManager::SetConnectionCallbacks(OnConnectionCompleteCallback connCompleteCb, OnConnectionClosedCallback connClosedCb,
```

Comment:
Apart from these methods, there are other snippets that are there which are TCP-specific in this file, and it would be difficult to extract all of those into a separate file.